src/components/question/question.js

```diff
@@ -22,12 +22,16 @@ class Question extends Component {
   render() {
     const offerButtonExists = this.props.openOfferModal ? this.renderOfferButton() : null ;
     return (
-      <div>
-        <p>Question: {this.props.question.title} Description: {this.props.question.description}</p>
+      <div className="question__container">
+        <div className="question__user">Avatar</div>
```

Comment:
Just wanted to add though you could use `.question__user:nth-child(1)` for the `question__user`
and use `.question__user:nth-child(2)` for `question__body` for differently styled children elements as well...but I think how you did it makes since to name them so you understand what they are in the styling sheet right away.